blog-pipeline/src/pipeline_smoke_test.py

```
@@ -42,20 +42,38 @@
+                _local_cfg = yaml.safe_load(_local.read_text()) or {}
+                def _deep_merge_cfg(base, override):
+                    result = dict(base)
+                    for k, v in override.items():
+                        if k in result and isinstance(result[k], dict) and isinstance(v, dict):
+                            result[k] = _deep_merge_cfg(result[k], v)
+                        else:
+                            result[k] = v
+                    return result
+                _pipeline_cfg = _deep_merge_cfg(_pipeline_cfg, _local_cfg)
             break
         except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)